packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -12,32 +12,20 @@ import {
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 
-export const StoryPromoLi = styled.li.attrs({
+export const StoryPromoLiThin = styled.li.attrs({
```

Comment:
This component appears to have a number of redundant styles/some that I would expect to be applied by the parent. Can we remove them?

Reply:
@simonsinclair which redundant styles do you mean ? the aim wasn't to get rid of any styles as we want the same behavior the story-promo-li gives us but with much more flexibility on how we apply spacing

Reply:
```
&:first-child {
    padding-top: 0;
```
What `padding-top: 0;`'s purpose on an element that already has no padding?

Reply:
😄 I agree, that needs to go